startup_runners/outer_startup_runner.c

```diff
@@ -13,44 +13,47 @@
 
 #include "libkruntime.h"
 
-#define BENCH_FUNC_NAME "run_iter"
+void emit_dummy_per_core_data(char *, int);
+
+void
+emit_dummy_per_core_data(char *name, int num_per_core_data)
+{
+    int i;
+
+    printf("\"%s\": [", name);
+    for (i = 0; i < num_per_core_data; i++) {
+        printf("[0, 0]");
+        if (i < num_per_core_data - 1) {
+            printf(", ");
+        }
+    }
+    printf("], ");
+}
 
 int
 main(int argc, char **argv)
 {
-    double start_time = -1;
-    int result;
-    char fake_datum[] = "[0, 0], ";
-    char end_datum[] = "[0, 0] ";
-    int datum_size = (int)strlen(fake_datum);
-    krun_init();
-    char* fake_data = malloc(1);
+    int result, num_per_core_data = -1;
```

Comment:
Why is `num_per_core_data` initialised but not `result`? I don't think either needs initialising here.